### PR TITLE
tasks for disabling automounting.

### PR DESCRIPTION
**Pull request by troy2914**

### CODE REVIEW

File: lastminutewf2020.yml [hunk 1]
```diff
@@ -75,3 +75,41 @@
...
+        group: root
+        mode: "0700"
+    - name: gvfs permission tweaking
+      ansible.builtin.file:
+        path: /usr/lib/gvfs/gvfs-{{ item }}
+        owner: root
+        group: root
+        mode: "0744"
+      loop:
+        - "udisks2-volume-monitor"
```
Comment (vmcj) [inline lastminutewf2020.yml:91]: Probably an item is missing? Otherwise the loop is not needed,

File: lastminutewf2020.yml [hunk 2]
```diff
@@ -75,3 +75,41 @@
         msg: "Finished {{ version }}"
         priority: "notice"
       changed_when: false
+    - name: workaround nautilus auto-mounting
```
Comment (vmcj) [inline lastminutewf2020.yml:78]: I think all of these should before the finished log entry on L75

File: lastminutewf2020.yml (hunk 1 above)
Comment (vmcj) [reply lastminutewf2020.yml:91]: Alternative is to add the gvfs in front and merge the loop here and below.

Comment (troy2914) [reply lastminutewf2020.yml]: hmm,  it was done this way,  because before there were move gvfs items.

File: lastminutewf2020.yml (hunk 2 above)
Comment (troy2914) [reply lastminutewf2020.yml:72]: done.

File: lastminutewf2020.yml (hunk 1 above)
Comment (troy2914) [reply lastminutewf2020.yml]: moved to a single loop

**Review by vmcj — APPROVED** [review]

**Pull request merged**